packages/terra-button-group/src/ButtonGroupButton.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { PropTypes } from 'react';
```

Comment:
The button group was originally just that, but as it became apparent that UX did not want all the available styles on button to be available for the button group, we ended up just creating a button group button with only those styles.